### DISCUSSION TITLE
Why when I make the window fullscreen, the border is seen on another monitor?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [hyprwm/Hyprland#5434] [Q&A]
[Video](https://files.catbox.moe/gvvi72.mp4)

**Comment by 0xFMD** [comment]
I think because of the spring animation. Try to open two windows and make the right one full screen

**Reply by azizsayfiddin** [reply]
Can I make the animations to not go out of the monitor?

**Reply by 0xFMD** [reply]
Maybe setting it to [linear](https://wiki.hyprland.org/Configuring/Animations/#curves) will do the job or turn off the animation.

here's mine (not linear BTW)
```
animations {
     enabled=1
 # bezier=overshot,0.05,0.9,0.1,1.1
    bezier=overshot,0.13,0.99,0.29,1.1
    animation=windows,1,4,overshot,slide
    animation=border,1,10,default
    animation=fade,1,10,default
    animation=workspaces,1,6,overshot,slidevert
}
```

**Reply by azizsayfiddin** [reply]
didn't help
